### ISSUE TITLE
(8/15) Yandex Page

### ISSUE THREAD

**Issue by bensternthal** [mozilla/bedrock#5635]
Success Criteria
- [ ] Use https://yandex.ru/firefox/mozilla as link to the Yandex build download page
- [x] Show the conditional content to users identified in Russia, only, based on geolocation
- [ ] Launch w. a/b test in place as described in [manapage](https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=80413405)

**Comment by sandboxjake70** [comment]
testing comments

**Comment by djst** [comment]
What is blocking this project?

**Comment by bensternthal** [comment]
On hold due to things still being worked out between Moz & Yandex. I will forward you an email chain with more information, out of our hands for now.

**Comment by alexgibson** [comment]
I just need a final copy deck and sketch files and then we're good to start dev work here.

**Comment by mrkwvr** [comment]
Final design sketch file
[Yandex Landing Page-07102018.sketch.zip](https://github.com/mozilla/bedrock/files/2181502/Yandex.Landing.Page-07102018.sketch.zip)

**Comment by alexgibson** [comment]
Link to final copy doc (in English): https://docs.google.com/document/d/1VsZsKD8wEzLJcaicg0I2uMYS0CZWr3V0ogACQ8CpY8E/edit#

**Comment by alexgibson** [comment]
I'm going to add a blocker to this task until we have the final Russian translation.

**Comment by alexgibson** [comment]
Other things we still need:

- The destination URL which we should point the dl button to (https://yandex.ru/firefox/mozilla is 404)
- The Yandex privacy policy link.

**Comment by ejregithub** [comment]
@alexgibson - just the FYI that https://yandex.ru/firefox/mozilla is now in place.

**Comment by alexgibson** [comment]
Localized copy: https://docs.google.com/document/d/1vL5BkSAuyIaIiPcNMyMznbCE_4Mha_nHIxKrbTJQzTE/edit

**Comment by alexgibson** [comment]
We’re still waiting on one last string translation, but here’s the updated demo with the Yandex copy in Russian https://bedrock-demo-agibson.oregon-b.moz.works/ru/firefox/new/?geo=ru (edited)

**Comment by alexgibson** [comment]
All strings are now in place, but we still need a GA & a fully formed test plan. Going to mark this as blocked.

**Comment by alexgibson** [comment]
With this new `/ru/firefox/new/` page, I'm going to simplify a couple of things given the new Yandex geolocation requirements.

- The page is no longer going to display an "up-to-date" or "out-of-date" notification bar at the top to existing Firefox users. We can't be sure exactly which version of FF Yandex will be providing on their download page, so I don't think we can safely assume what we can consider "up to date" since this information is not in product details.
- The page will no longer display a "Refresh Firefox" button. We don't want this to compete with the Yandex CTA, but also since we're referring users to another site to download Firefox that comes with pre-installed add-ons, presenting users with a button that removes all those add-ons may not be something those users expect. I think it may be best to refer people to the Yandex site here still.

Thoughts, @ejregithub ?

**Comment by alexgibson** [comment]
Analytics requirements: https://docs.google.com/document/d/1zGfVM6nn1xgFtvf4ri8STgN1PXRVziJBNLu8Az91x2Q/edit?usp=sharing
